ably/__init__.py

```diff
@@ -1,4 +1,5 @@
 from ably.rest.rest import AblyRest
+from ably.realtime.realtime import AblyRealtime
```

Comment:
I realise the same is present in the previous line (with `rest.rest`) but I have to admit that `realtime.realtime` looks a bit odd. I assume, without much Python knowledge, that this comes from it being from a file called `realtime.py` within the folder called `realtime`.

Perhaps we could consider renaming the `py` file to something which adds more meaning - perhaps `main`, `default`, `core`? Alternatively, could we consider moving this file up a folder?

This might be entirely idiomatic for Python, of course, so feel free to ignore me. It just feels a bit noisy in respect of namespace.

Reply:
I think this one is fine to leave, `ably.realtime.realtime` is just the internal directory structure so you could read it as `ably/realtime/realtime.py`. I can see that it looks a little bit odd but I think the way project is structured at the moment is sensible and would require a fair bit of effort to change.

Reply:
ok, thanks for explaining.